dp/init.c

```diff
@@ -302,14 +300,12 @@ void dp_port_init(void)
 {
 #ifndef USE_AF_PACKET
 	uint8_t port_id;
+	int i = 0, j; //GCC_Security flag
```

Comment:
i think they added = 0 to prevent warning -> error in GCC. Should that be done to `j` too? How come this didn't complain

Reply:
This was resolved after discussion.